app/scripts/views/permissions.js

```diff
@@ -30,20 +71,162 @@ define(function (require, exports, module) {
 
     context: function () {
       var account = this.getAccount();
+      var requestedPermissions = this.relier.get('permissions');
+      var appliciblePermissions =
+        this._getApplicablePermissions(account, requestedPermissions);
+      var permissionsHTML = this._getPermissionsHTML(account, appliciblePermissions);
+
       return {
-        email: account.get('email'),
+        displayName: account.get('displayName') || 'None set',
         privacyUri: this.relier.get('privacyUri'),
         serviceName: this.relier.get('serviceName'),
-        termsUri: this.relier.get('termsUri')
+        termsUri: this.relier.get('termsUri'),
+        unsafePermissionsHTML: permissionsHTML,
       };
     },
 
+    /**
+     * Get configuration for a permission
+     *
+     * @private
+     * @param {string} permissionName
+     * @returns {object} permission, if found.
+     * @throws if permission is invalid
+     */
+    _getPermissionConfig: function (permissionName) {
+      var permission = _.findWhere(PERMISSIONS, { name: permissionName });
+
+      if (! permission) {
```

Comment:
This behavior could cause unexpected behavior with current reliers. Let's do a pass at stricter scope checking in a separate PR. Silently ignore the permission for now.